test/ReactSetStateTestComponent.js

```diff
@@ -27,15 +27,28 @@ var ReactSetStateTestComponent = React.createClass({
     this.recordLifecyleCall("componentWillReceiveProps");
   },
 
+  componentWillReceivePropsWithContext: function(_) {
```

Comment:
These `-WithContext` methods on the JS component won't ever get called, right? Why do they exist?

Reply:
I believe it was only testing so `recordLifecyleCall` would record the call and could be validated.

Reply:
Not sure I'm following. `recordLifecycleCall` won't be called since nothing is calling these methods on the JS component, right?